internetarchive/item.py

```diff
@@ -953,11 +954,20 @@ def _build_request():
                     expected_size = math.ceil(size / chunk_size)
                     chunks = chunk_generator(body, chunk_size)
                     progress_generator = tqdm(chunks,
-                                              desc=f' uploading {key}',
-                                              dynamic_ncols=True,
-                                              total=expected_size,
-                                              unit='MiB')
-                    data = IterableToFileAdapter(progress_generator, size)
+                                          desc=f' uploading {key}',
+                                          dynamic_ncols=True,
+                                          total=expected_size,
+                                          unit='MiB')
```

Comment:
Style nit: leave the indentation of the `tqdm` arguments as it was.

Reply:
Let’s have machines do code formatting so people do not have to.  #532